harmonica/_forward/point.py

```diff
@@ -448,31 +309,39 @@ def kernel_potential_spherical(
     distance, _, _ = distance_spherical_core(
         longitude, cosphi, sinphi, radius, longitude_p, cosphi_p, sinphi_p, radius_p
     )
-    return 1 / distance
+    return 1 / distance * GRAVITATIONAL_CONST
 
 
 #  Acceleration components
 #  -------------------
 
 
 @jit(nopython=True)
-def kernel_g_z_spherical(
+def kernel_gravity_u_spherical(
     longitude, cosphi, sinphi, radius, longitude_p, cosphi_p, sinphi_p, radius_p
 ):
     """
-    Kernel for downward component of gravitational acceleration
+    Kernel for upward component of gravitational acceleration
 
     Use spherical coordinates
     """
     distance, cospsi, _ = distance_spherical_core(
         longitude, cosphi, sinphi, radius, longitude_p, cosphi_p, sinphi_p, radius_p
     )
     delta_z = radius - radius_p * cospsi
-    return delta_z / distance**3
+    return -GRAVITATIONAL_CONST * delta_z / distance**3
```

Comment:
Since now these functions return the actual gravity field of a single point mass, shouldn't we rename them to `gravity_u_spherical`?

I know that we are going to implement them in Choclo and these will be gone soon, but this could be an opportunity to start drafting how they are going to be used in Harmonica.

Reply:
> Member

Yup~ Good point 👍